core-ui/src/components/Predefined/Details/PersistentVolumeClaims/PersistentVolumeClaims.details.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { PersistentVolumeClaimStatus } from 'shared/components/PersistentVolumeClaimStatus';
+
+import {
+  EMPTY_TEXT_PLACEHOLDER,
+  GenericList,
+  navigateToFixedPathResourceDetails,
+  Labels,
+  useGetList,
+} from 'react-shared';
+import { LayoutPanel, Link } from 'fundamental-react';
+import { LayoutPanelRow } from 'shared/components/LayoutPanelRow/LayoutPanelRow';
+import { Tokens } from 'shared/components/Tokens';
+import { RelatedPods } from './RelatedPods';
+
+const PVCSelectorSpecification = pvc => {
+  const { t, i18n } = useTranslation();
+  const headerRenderer = () => [
+    t('persistent-volume-claims.headers.key'),
+    t('persistent-volume-claims.headers.operator'),
+    t('persistent-volume-claims.headers.values'),
+  ];
+
+  const rowRenderer = ({ key = '', operator = '', values = [] }) => [
+    key,
+    operator,
+    <Tokens tokens={values} />,
+  ];
+  return (
+    <>
+      <LayoutPanel className="fd-margin--md" key={'pvc-selector'}>
+        <LayoutPanel.Header>
+          <LayoutPanel.Head title="Selector" />
+        </LayoutPanel.Header>
+        <LayoutPanel.Body>
+          <LayoutPanelRow
+            name={t('persistent-volume-claims.headers.match-labels')}
+            value={
+              <Labels labels={pvc.spec?.selector?.matchLabels} /> ||
+              EMPTY_TEXT_PLACEHOLDER
+            }
+          />
+        </LayoutPanel.Body>
+      </LayoutPanel>
+      <GenericList
+        title={t('persistent-volume-claims.headers.match-expressions')}
+        entries={pvc.spec?.selector?.matchExpressions || []}
+        headerRenderer={headerRenderer}
+        rowRenderer={rowRenderer}
+        testid="daemon-set-tolerations"
+        i18n={i18n}
+        showSearchField={false}
+      />
+    </>
+  );
+};
+
+export const PVCConfiguration = pvc => {
+  const { t } = useTranslation();
+
+  const { data: storageClasses } = useGetList()(
+    '/apis/storage.k8s.io/v1/storageclasses',
+  );
+  return (
+    <LayoutPanel className="fd-margin--md" key={'pvc-configuration'}>
+      <LayoutPanel.Header>
+        <LayoutPanel.Head title="Configuration" />
+      </LayoutPanel.Header>
+      <LayoutPanel.Body>
+        <LayoutPanelRow
+          key={pvc.spec.volumeMode}
+          name={t('persistent-volume-claims.headers.volume-mode')}
+          value={pvc.spec.volumeMode}
+        />
+        <LayoutPanelRow
+          key={pvc.spec.accessModes}
+          name={t('persistent-volume-claims.headers.access-modes')}
+          value={<Tokens tokens={pvc.spec.accessModes} />}
+        />
+        <LayoutPanelRow
+          key={pvc.spec?.volumeName}
+          name={t('persistent-volume-claims.headers.volume-name')}
+          value={
+            pvc.spec?.volumeName ? (
+              <Link
+                onClick={() =>
```

Comment:
afaik, storage classes are not namespace-wide, but cluster-wide, so we need to adjust this navigation.